client/src/components/Data/FeedbackData.ts

```diff
@@ -0,0 +1,31 @@
+import { Feedback } from 'types';
+
+export const FeedbackData: Feedback[] = [
+  {
+    id: '1',
+    nickname: 'Anna',
+    email: 'anna@mail.com',
+    title: 'great',
+    review:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. consequuntur vel vitae commodi alias voluptatem est voluptatum ipsa quae.',
+    rating: 5
+  },
+  {
+    id: '2',
+    nickname: 'Jon',
+    email: 'jon@mail.com',
+    title: 'normal',
+    review:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. consequuntur vel vitae commodi alias voluptatem est voluptatum ipsa quae.',
+    rating: 4
+  },
+  {
+    id: '3',
+    nickname: 'Kris',
+    email: 'kris@mail.com',
+    title: 'works',
+    review:
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. consequuntur vel vitae commodi alias voluptatem est voluptatum ipsa quae.',
+    rating: 3
+  }
+];
```

Comment:
When you or any other user opens this page, they see nothing except form inputs because their localStorage is empty. To understand better the design and how everything works, I made the mock data.

Reply:
it is easier to work, because you don't need to submit form many times

Reply:
You can add some feedbacks in LS. When you open this page, you can get data from LS. 
I think in real case app will use database and all users will see feedbacks from DB